### PR TITLE
Improve admin page

### PR DESCRIPTION
**Pull request by abcang**
* Improve report page.
  * Add batch action (change NSFW or delete status),
  * Added function to show and hide all media.
  * Changed to delete status asynchronously.
* Add account status page.
  * Delete status and change NSFW possible.
  * There is a filter to narrow down the media.

Report page:
image

Account status page:
image

Account page:
image

### CODE REVIEW

File: app/controllers/admin/reported_statuses_controller.rb [hunk 1]
```diff
@@ -5,7 +5,15 @@ class ReportedStatusesController < BaseController
...
 
     before_action :set_report
-    before_action :set_status
+    before_action :set_status, only: [:update, :destroy]
+
+    def create
+      @form = Form::StatusBatch.new(form_status_batch_params)
+      unless @form.save
+        flash[:alert] = t('admin.statuses.failed_to_execute')
+      end
```
Comment (ykzts) [inline app/controllers/admin/reported_statuses_controller.rb]: `flash[:alert] = t('admin.statuses.failed_to_execute') unless @form.save`

File: app/controllers/admin/statuses_controller.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      @form = Form::StatusBatch.new
+    end
+
+    def create
+      @form = Form::StatusBatch.new(form_status_batch_params)
+      unless @form.save
+        flash[:alert] = t('admin.statuses.failed_to_execute')
+      end
+      redirect_to admin_account_statuses_path(@account.id, current_params)
+    end
```
Comment (ykzts) [inline app/controllers/admin/statuses_controller.rb]: `flash[:alert] = t('admin.statuses.failed_to_execute') unless @form.save`

File: app/controllers/admin/reported_statuses_controller.rb [hunk 3]
```diff
@@ -15,7 +23,7 @@ def update
     def destroy
       authorize @status, :destroy?
       RemovalWorker.perform_async(@status.id)
-      redirect_to admin_report_path(@report)
+      render json: @status
```
Comment (ykzts) [inline app/controllers/admin/reported_statuses_controller.rb]: Also fix [`spec/controllers/admin/reported_statuses_controller_spec.rb`](https://github.com/abcang/mastodon/blob/7fa0897d3518a1338e042342c6c2a686e708fd27/spec/controllers/admin/reported_statuses_controller_spec.rb#L47-L54).

**Review by ykzts — DISMISSED** [review]

**Comment by abcang** [comment]
I Improved select style

**Review by beatrix-bitrot — APPROVED** [review]
nice work!! LGTM

File: config/locales/en.yml [hunk 4]
```diff
@@ -188,6 +188,21 @@ en:
         title: Site Privacy Policy
       site_title: Site title
       title: Site Settings
+    statuses:
+      back_to_account: Back to account page
+      batch:
+        delete: Delete
+        nsfw_off: NSFW OFF
+        nsfw_on: NSFW ON
```
Comment (m4sk1n) [inline config/locales/en.yml]: NSFW? Isn't it now called 'Content warning' on Mastodon?

Comment (abcang) [inline config/locales/en.yml]: "Content warning" is a function to hide sentences, and NSFW is a function to hide images. The implemented function is a function to change NSFW.

**Pull request merged**